Makefile

```diff
@@ -60,7 +60,7 @@ SWAGGER_IMAGE=quay.io/goswagger/swagger:$(SWAGGER_VER)
 SWAGGER=docker run --rm -it -u ${shell id -u} -v "${PWD}:/go/src/" -w /go/src/ $(SWAGGER_IMAGE)
 SWAGGER_GEN_DIR=swagger-gen
 
-JAEGER_DOCKER_PROTOBUF=jaegertracing/protobuf:0.2.0
+JAEGER_DOCKER_PROTOBUF=otel/build-protobuf:0.2.1
```

Comment:
Could we make the corresponding upgrade in our `protobuf` image?

Alternatively, are there significant differences between them? Maybe it would make sense to deprecate ours altogether.

Reply:
Why did we stop using the upstream image? The last time I've checked, the image was well-maintained.

https://github.com/TheThingsIndustries/docker-protobuf

Reply:
We never used upstream image. It doesn't include gogo iirc.

Reply:
Trying to use the upstream image (`thethingsindustries/protoc:3.1`) fails with 

> protoc-gen-swagger/options/annotations.proto: File not found.
> query.proto:25:1: Import "protoc-gen-swagger/options/annotations.proto" was not found or had errors.

We can upgrade our image if we want.  I'm fine with either option, but I figured it was worth discussing.  Here is the diff to upgrade `jaegertracing/protobuf`:

https://gist.github.com/joe-elliott/249686fa8329d7700b1044bff5a34aee

This was largely taken from the otel protobuf image.

Reply:
Let's upgrade ours, can you create a PR?